src/Raven.Server/Monitoring/Snmp/Objects/Database/5.2.X/1/DatabaseCountOfStaleIndexes.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using Lextm.SharpSnmpLib;
+using Raven.Server.Documents;
+using Raven.Server.ServerWide.Context;
+
+namespace Raven.Server.Monitoring.Snmp.Objects.Database
+{
+    public class DatabaseCountOfStaleIndexes : DatabaseScalarObjectBase<Gauge32>
+    {
+        public DatabaseCountOfStaleIndexes(string databaseName, DatabasesLandlord landlord, int index)
```

Comment:
Do we verify that the same db name is under the same index in all servers?

Reply:
No, monitoring is per server, not sure if we should mix cluster in this?

Reply:
From the other side it could be more convenient to have unified indexes...

Reply:
Yes, that is my thinking as well. If we are monitoring a cluster of system, it would be nice if we had it matching. We can use a hash of the name, I guess?